.github/workflows/native-ubuntu.yml

```diff
@@ -0,0 +1,135 @@
+name: Native Ubuntu
+
+on:
+  push:
+    branches: [ main, ci-sandbox ]
+  pull_request:
+    branches: [ main, ci-sandbox ]
+
+jobs:
+  build-ubuntu:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "full-marian"
+            os: ubuntu-latest
+            gcc: 8
+            cpu: 'ON'
+            gpu: 'OFF'
+            test_tags: ""
+            cmake: 
+              CMAKE_BUILD_TYPE: "Release"
+              COMPILE_TESTS: "ON"
+              USE_WASM_COMPATIBLE_SOURCE: "OFF"
+              COMPILE_SERVER: "OFF"
+              COMPILE_EXAMPLES: "OFF"
+
+          - name: "full-marian-slim"
+            os: ubuntu-latest
+            gcc: 8
+            cpu: 'ON'
+            gpu: 'OFF'
+            test_tags: ""
+            cmake: 
+              CMAKE_BUILD_TYPE: "Slim"
+              COMPILE_TESTS: "ON"
+              USE_WASM_COMPATIBLE_SOURCE: "OFF"
+              COMPILE_SERVER: "OFF"
+              COMPILE_EXAMPLES: "OFF"
+
+          - name: "minimal-marian"
+            os: ubuntu-latest
+            gcc: 8
+            cpu: 'ON'
+            gpu: 'OFF'
+            test_tags: "'#wasm'"
+            cmake:
+              CMAKE_BUILD_TYPE: "Release"
+              COMPILE_TESTS: "OFF" # Minimal marian has no sqlite support and COMPILE_TEST=ON fails.
```

Comment:
@jerinphilip One final ask. Could you please add USE_STATIC_LIBS: "ON" here as well? It will make both mac and ubuntu workflow files consistent. It is a nit so you can ignore it as well.

Rest of the PR looks great. I can approve it 👍

Reply:
So mac was the exception, because of the BLAS issue. The default is USE_STATIC_LIBS: ON in the repo, and ubuntu workflows don't use the switch. I can add a note if necessary, as this key is not used anywhere. The note is supposed to exist for Mac I think which is the exception due to BLAS issues. For Mac, I  believe the minimal marian is circumventing this in the other mode using supplied BLAS at the moment.

https://github.com/browsermt/bergamot-translator/blob/2e5daac978f535b14bf32997e563f3bf430efbfb/CMakeLists.txt#L24

Reply:
Got it.